jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -76,6 +76,12 @@ public StepsRunner(
     this.sourceFilesConfiguration = sourceFilesConfiguration;
     this.baseLayersCache = baseLayersCache;
     this.applicationLayersCache = applicationLayersCache;
+
+    ExecutorService executorService =
+        System.getProperty("jibSerialized") == null
```

Comment:
Should we check for a specific value of `jibSerialized` for the serialized case (to avoid someone accidentally using serialized when not intentional)?

Reply:
Turned it into a boolean property that must be `true`.